README.md

```diff
@@ -42,6 +42,7 @@ export LOADTESTING_API_KEY='FIRETEXT_SIMULATION_KEY'
 export FIRETEXT_API_KEY='FIRETEXT_ACTUAL_KEY'
 export STATSD_PREFIX='YOU_OWN_PREFIX'
 export NOTIFICATION_QUEUE_PREFIX='YOUR_OWN_PREFIX'
+export REDIS_URL="redis://localhost:6379/0"
```

Comment:
should probably mention just below, after postgres, that you can install redis locally, but will need to change your local config if you want to use it